File: Kotlin/03_conditionals.kt
fun main() {
    val age = 20

    val result = when {
        age < 18 -> "Minor"
        age in 18..64 -> "Adult"
        else -> "Senior"
    }
    println(result)

    val status = "active"
    println(when (status) {
        "active" -> "User is active"
        else -> "Unknown"
    })
}

File: Cpp/03_conditionals.cpp
#include <iostream>

int main() {
    int age = 20;

    if (age < 18) {
        std::cout << "Minor" << std::endl;
    } else if (age >= 18 && age < 65) {
        std::cout << "Adult" << std::endl;
    } else {
        std::cout << "Senior" << std::endl;
    }

    // Switch (C++ has no built-in match yet – switch is the classic)
    std::string status = "active";
    switch (status[0]) {  // simple demo
        case 'a': std::cout << "User is active" << std::endl; break;
        default:  std::cout << "Unknown" << std::endl;
    }
    return 0;
}
